tracing/opentracing/id_extract.go

```diff
@@ -38,5 +40,14 @@ func (t *hackyTagsCarrier) Set(key, val string) {
 		t.Tags.Set(TagTraceId, val) // this will most likely be base-16 (hex) encoded
 	} else if (strings.Contains(key, "spanid") && !strings.Contains(key, "parent")) || (strings.Contains(strings.ToLower(key), "spanid") && !strings.Contains(strings.ToLower(key), "parent")) {
 		t.Tags.Set(TagSpanId, val) // this will most likely be base-16 (hex) encoded
+	} else if key == "uber-trace-id" {
+		parts := strings.Split(val, ":")
+		if len(parts) == 1 {
+			// for extraordinary cases
```

Comment:
Extraordinary cases? Maybe special cases?

Reply:
Under what circumstance would this happen? (I'm not very familiar with Jaeger)

Reply:
I have not encountered such cases, so I left this comment

Reply:
Is it ok if only normal case would be handled?